### ISSUE TITLE
[Bug]: Login passwordless on Android no NFC accepted

### ISSUE THREAD

**Issue by chrismaster** [zitadel/zitadel#7363]
### Preflight Checklist

- [X] I could not find a solution in the documentation, the existing issues or discussions
- [ ] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Environment

Self-hosted

### Version

2.44.2

### Database

CockroachDB

### Database Version

23.2

### Describe the problem caused by this bug

Login passwordless login doesn't show NFC, only USB on Android. So no passwordless login with NFC is possible on Android.
2FA auth with NFC on Android works....

### To reproduce

Go to Login Page
Test with passwordless login on Android over NFC.

**Comment by fforootd** [comment]
Hi, can you share us some details.

What OS and Browser type/version are you using?

**Comment by chrismaster** [comment]
Android v14 with Google Play-Systemupdate 1.November 2023
Chrome v 121.0.6167.164 from 6.2.2024
@ Samsung A34

**Comment by fforootd** [comment]
Thanks for sharing this. I have not yet been able to reproduce this 😓 

Can anybody with android give this a spin and see if NFC as passwordless auth works?

Calling my fellow android users to help 😁 @peintnermax @hifabienne @mffap

**Comment by mffap** [comment]
> Thanks for sharing this. I have not yet been able to reproduce this 😓
> 
> Can anybody with android give this a spin and see if NFC as passwordless auth works?
> 
> Calling my fellow android users to help 😁 @peintnermax @hifabienne @mffap

Are we discussing U2F?
Works on Pixel 6 / Android 14 it seems

![Screenshot_20240227-092611~2](https://github.com/zitadel/zitadel/assets/1366906/87212bed-0e40-4949-b104-74447475213e)

**Comment by fforootd** [comment]
> Are we discussing U2F?

I think it was about FIDO2 / Passkeys / CTAP 2

**Comment by hifabienne** [comment]
I tested both and can reproduce.
![Screenshot_20240228-074803.png](https://github.com/zitadel/zitadel/assets/38692350/5d2b4aaf-6c79-43e2-beea-4fbdc3a209b0)

![Screenshot_20240228-074818.png](https://github.com/zitadel/zitadel/assets/38692350/22069fc7-c4dd-4c48-bb58-517408fd9e2a)

**Comment by fforootd** [comment]
Ok so it is a bug 😁

**Comment by livio-a** [comment]
@muhlemmer will check again. It could be that U2F works since this is a single factor. In case of Passekeys / CTAP2 it will be jandled as a MFA and therefore requires user verification / PIN. In case the platform does not support this it will not provide the NFC option.

**Comment by muhlemmer** [comment]
Doesn't seem to be a zitadel bug, but rather a platform of browser bug. I've set up a yubikey 5 NFC passkey with github and it doesn't show the option there either:

![github in chrome for android, no passkey over NFC option](https://github.com/zitadel/zitadel/assets/5411563/c0f314ba-4870-415f-a825-12291208e56c)

As an alternative I've tried Firefox for android. The option shows, but the action fails without ever asking for the yubikey PIN:

![github in firefox for android, passkey over NFC fails](https://github.com/zitadel/zitadel/assets/5411563/35f8ae87-fe95-4f7c-b15e-1a869f105011)

Same goes for trying it with zitadel on firefox for android, again the yubikey PIN was never requested:

![zitadel in firefox for android, passkey over NFC fails](https://github.com/zitadel/zitadel/assets/5411563/0f3219d7-2efc-4c22-a3a9-fb8fbbdb98c2)

Don't know if both situations are caused by the same bug (that means the android platform or yubikey has the fault) or seperate bugs for each browser.

At least after some googling, I found an open issue at chromium:

https://issues.chromium.org/issues/328710551
